static/app/components/onboarding/frameworkSuggestionModal.tsx

```diff
@@ -56,25 +58,43 @@ const topNodeFrameworks = [
   'node-koa',
 ];
 
+const topDotNetFrameworks = [
+  'dotnet-aspnetcore',
+  'dotnet-aspnet',
+  'dotnet-maui',
+  'dotnet-wpf',
+  'dotnet-winforms',
+  'dotnet-xamarin',
+  'dotnet-uwp',
+  'dotnet-gcpfunctions',
+  'dotnet-awslambda',
```

Comment:
@bruno-garcia  also mentioned standard csharp in [the list](https://github.com/getsentry/sentry/issues/51903) - is it equivalent to skipping the modal and choosing the vanilla?

Reply:
yes!